homeassistant/components/samsungtv/bridge.py

```diff
@@ -383,6 +383,13 @@ def _get_device_spec(self, key: str) -> Any | None:
 
     async def async_is_on(self) -> bool:
         """Tells if the TV is on."""
+        # On some TVs, opening a websocket turns on the TV
+        # so first check existing websocket connection
+        # then fallback to "PowerState" if device_info has it
+        # then fallback to default, trying to open a websocket
+        if (remote := self._remote) and remote.is_alive():
+            LOGGER.debug("Checked that TV %s is on using existing websocket", self.host)
+            return True
```

Comment:
I think we still need to check the powerstate if the websocket is connected because the tv can be turned off manually and the websocket stays active for a minutes or so

Reply:
In that case - let's just add a comment in the code

Reply:
I've updated the PR description accordingly